src/Components/Publishing/Fixtures/Articles.ts

```diff
@@ -669,15 +669,29 @@ export const MissingVerticalStandardArticle = extend({}, StandardArticle, {
   vertical: null,
 })
 
+export const VideoArticle = extend({}, StandardArticle, {
```

Comment:
This is minor, but it seems a little odd to extend another article layout in this instance since it means `VideoArticle` also has `sections` which shouldn't be true. I would prefer to create a new one or make a secondary Generic article with just the basics.